Source/MethodCall.cs

```diff
@@ -81,42 +81,36 @@ public IVerifies Raises(Action<TMock> eventExpression, params object[] args)
 		}
 	}
 
-	internal class TypeEqualityComparer : IEqualityComparer<Type>
-	{
-		public bool Equals(Type x, Type y)
-		{
-			return y.IsAssignableFrom(x);
-		}
-
-		public int GetHashCode(Type obj)
-		{
-			return obj.GetHashCode();
-		}
-	}
-
 	internal partial class MethodCall : IProxyCall, ICallbackResult, IVerifies, IThrowsResult
 	{
-		// Internal for AsMockExtensions
-		private Expression originalExpression;
-		private Exception thrownException;
-		private Action<object[]> setupCallback;
-		private List<IMatcher> argumentMatchers = new List<IMatcher>();
-		private EventInfo mockEvent;
-		private Delegate mockEventArgsFunc;
-		private object[] mockEventArgsParams;
+		private List<IMatcher> argumentMatchers;
+		private Action<object[]> callbackResponse;
+		private int callCount;
+		private Condition condition;
 		private int? expectedMaxCallCount;
```

Comment:
This could probably be turned into an `int` and initialized to `int.MaxValue`, let's not for the moment.